src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/Report.kt

```diff
@@ -437,18 +437,18 @@ class Report(
     History(
       report = this,
       type = Type.fromNomisCode(nomisHistory.type),
-      changedAt = nomisHistory.incidentChangeDate.atStartOfDay(),
+      changedAt = nomisHistory.createDateTime,
```

Comment:
Switched for now to use the create date time as this contains a time portion

Reply:
should this not be `nomisHistory.incidentChangeDateTime ?: nomisHistory.createDateTime`